src/components/DeprecatedBanner/DeprecatedBanner.js

```diff
@@ -22,6 +23,12 @@ const DeprecatedBanner = props => (
 )
 
 class DeprecatedDao extends React.Component {
+  static propTypes = {
+    dao: PropTypes.object.isRequired,
```

Comment:
From what I can tell, this should actually just be a string (either an address or an ENS name)?